src/Services/ObservationLabService.php

```diff
@@ -38,6 +44,143 @@ public function __construct()
         (new UuidRegistry(['table_name' => self::PATIENT_TABLE]))->createMissingUuids();
     }
 
+    public function getUuidFields(): array
+    {
+        return ['uuid', 'puuid'];
+    }
+
+    private function getSampleLaboratoryResults()
+    {
+        $factory = new \Ramsey\Uuid\UuidFactory();
```

Comment:
would include this in use block above just to make easier to track class use

Reply:
this should be fixed now.